vaccine_feed_ingest/runners/us/giscorps_vaccine_providers/normalize.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+
+SOURCE_NAME = "us_giscorps_vaccine_providers"
+
+
+def _get_availability(site: dict) -> schema.Availability:
+    appt_only = site["attributes"]["appt_only"]
+    if appt_only == "Yes":
+        return schema.Availability(appointments=True)
+    elif appt_only is not None:
+        return schema.Availability(drop_in=True)
```

Comment:
Are we confident that "walk-in" should be the default value for a location that is not `appt_only` (vs. unknown)

Reply:
was not aware unknown was an option. will update